src/components/screens/wallet/overview/index.js

```diff
@@ -53,24 +111,32 @@ const Overview = ({
           isMultisignature={isMultisignature}
         />
       </div>
-      <div className={`${grid['col-xs-6']} ${grid['col-md-3']} ${grid['col-lg-3']}`}>
+      <div
+        className={`${grid['col-xs-6']} ${grid['col-md-3']} ${grid['col-lg-3']}`}
+      >
         <BalanceInfo
           t={t}
           activeToken={activeToken}
           balance={balance}
-          isDiscreetMode={discreetMode}
+          isDiscreteMode={discreteMode}
           isWalletRoute={isWalletRoute}
           username={account?.dpos?.delegate?.username}
           address={address}
+          isBanned={isBanned}
+          pomStart={pomHeights?.length
+            ? { ...pomHeights[pomHeights.length - 1] }
+            : {}}
         />
       </div>
-      <div className={`${grid['col-xs-12']} ${grid['col-md-6']} ${grid['col-lg-6']} ${styles.balanceChart}`}>
+      <div
+        className={`${grid['col-xs-12']} ${grid['col-md-6']} ${grid['col-lg-6']} ${styles.balanceChart}`}
+      >
         {address && (
           <BalanceChart
             t={t}
             transactions={isWalletRoute ? confirmed : transactions.data.data}
             token={activeToken}
-            isDiscreetMode={discreetMode && host === address}
+            isDiscreteMode={discreteMode && host === address}
```

Comment:
Discreet means cautious, while discrete means divided. Please  revert this change everywhere.